core/controllers/profile.py

```diff
@@ -77,6 +79,17 @@ def get(self, username):
         if not user_settings:
             raise self.PageNotFoundException
 
+        user_contributions = user_services.get_user_contributions(
+            user_settings.user_id)
+
+        created_exploration_summaries = (
+            exp_services.get_displayable_exploration_summaries_matching_ids(
+                user_contributions.created_exploration_ids))
+
+        edited_exploration_summaries = (
+            exp_services.get_displayable_exploration_summaries_matching_ids(
```

Comment:
I don't think that will work; exp_summary is a Python object. Look at other places in the codebase to see where it's done -- I think you'll need to enumerate the fields explicitly, actually, since you don't need all the information from the summary object. Just send to the frontend only the fields that the frontend needs.